skll/experiments/utils.py

```diff
@@ -84,13 +84,15 @@ def _create_learner_result_dicts(
     ----------
     task_results : List[EvaluateTaskResults]
         The task results list.
-    grid_scores : List[float]
-        The grid scores list.
-    grid_search_cv_results_dicts : List[Dict[str, Any]]
+    grid_scores : Union[List[None], List[float]]
+        The grid scores list or a list containing `None` instances for tasks
```

Comment:
Suggestion:
A list of grid scores or a list of `None` for tasks that do not involve grid search

Reply:
You know you can add an actual suggestion using the "diff" (plus/minus) icon? 😉